SeleniumTestXUnit/Core/Drivers/ChromeWebDriver.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace SeleniumTest.Core.Drivers;
+
+public class ChromeWebDriver
+{
+    public IWebDriver Driver { get; }
+    private readonly ConfigBuilder _config = ConfigBuilder.Instance;
+
+    public ChromeWebDriver()
+    {
+        Driver = new ChromeDriver();
+        Driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(
+            _config.GetInt("DriverImplicitTimeout")
```

Comment:
You can implement another class that wraps the ConfigBuilder like this
``` csharp
public class Env
{
  // initialize _config
  public int DriverImplicitTimeout = _config.GetInt("DriverImplicitTimeout");
  // similar with other env vars
}
```
This way you won't have to remember to write the env var names correctly, the IDE will tell you which env var are available to you through the Env, it would be awesome if you can make it static even, not sure if that's possible though